pandas/io/formats/format.py

```diff
@@ -1132,20 +1132,39 @@ def write_tr(self, line, indent=0, indent_delta=4, header=False,
         self.write('</tr>', indent)
 
     def write_style(self):
-        template = dedent("""\
-            <style>
-                .dataframe thead tr:only-child th {
-                    text-align: right;
-                }
-
-                .dataframe thead th {
-                    text-align: left;
-                }
-
-                .dataframe tbody tr th {
-                    vertical-align: top;
-                }
-            </style>""")
+        template_first = """\
+            <style scoped>"""
```

Comment:
maybe add a small comment (not inside the string but a python comment above `temlate_first` definition) on why the `scoped` is needed (as I think this could be easily overlooked when somebody else would later refactor this code)

Reply:
Comment added.